src/components/notification/RegularNotification.jsx

```diff
@@ -55,33 +56,83 @@ const styles = StyleSheet.create({
   user: {
     fontWeight: 'bold',
   },
+  buttonContainer: {
+    flex: 1,
+    height: 70,
+    width: 75,
+    backgroundColor: 'rgba(255,255,255,1)',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  acceptButton: {
+    backgroundColor: 'rgba(63,114,155,1)',
+    padding: 5,
+    borderRadius: 5,
+    marginTop: 8,
+    marginBottom: 4,
+  },
+  rejectButton: {
+    backgroundColor: 'rgba(237,92,69,1)',
+    padding: 5,
+    borderRadius: 5,
+    margin: 5,
+  },
+  buttonText: {
+    fontSize: 11,
+    color: 'white',
+    fontWeight: 'bold',
+  },
 });
 
 function RegularNotification({
   user, notificationType, userImage, notificationImage,
 }) {
   let notificationText = '';
+
   switch (notificationType) {
     case 'followed':
-      notificationText = 'has followed you bro.';
+      notificationText = 'has followed you bro';
       break;
     case 'joined':
-      notificationText = 'has joined your super cool event man good stuff.';
+      notificationText = 'has joined your super cool event man good stuff';
       break;
     case 'lit':
-      notificationText = 'has set an event to the lit stage.';
+      notificationText = 'has set an event to the lit stage';
       break;
     case 'completed':
       notificationText = 'has set an event to the completed stage';
       break;
+    case 'friend':
+      notificationText = 'is requesting to be your friend';
+      break;
     default:
-      notificationText = 'is showing you the default notification.';
+      notificationText = 'is showing you the default notification';
       break;
   }
+  if (notificationType === 'followed' || notificationType === 'joined' || notificationType === 'lit' || notificationType === 'completed') {
```

Comment:
Check for friend type, and add a default error case. You could also use a switch case here, like:

```
switch (notificationType) {
  case 'followed':
  case 'joined':
  case 'lit':
  case 'completed':
    return { ... }
    break;
  case 'friend':
    return { ... }
  default:
    return { ... }
}
```

Reply:
good idea.